src/dftbp/dftb/rshgamma.F90

```diff
@@ -456,31 +483,31 @@ pure function getYGammaSubPart(tauA, tauB, R, omega) result(yGamma)
     real(dp), intent(in) :: tauB
 
     !> Separation of the sites A and B
-    real(dp), intent(in) :: R
+    real(dp), intent(in) :: dist
 
     !> Range-separation parameter
     real(dp), intent(in) :: omega
 
     !> Resulting off-site Y-Gamma-integral
     real(dp) :: yGamma
 
-    !!
+    !! Auxiliary variables
```

Comment:
Let's simply remove this comment, as it does not state anything useful...

Reply:
done